### ISSUE TITLE
Analysis Exception introduced with Python3 (Attempt to get location)

### ISSUE THREAD

**Issue by Ana06** [vivisect/vivisect#359]
Vivisect doesn't extract the same features for this binary with the Python 3 (master) and with Python 2 (vivisect 0.2.0):
[https://github.com/fireeye/capa-testfiles/blob/master/9ff8e68343cc29c1036650fc153e69f7.exe_](https://github.com/fireeye/capa-testfiles/blob/master/9ff8e68343cc29c1036650fc153e69f7.exe_)

I think it is caused by this exceptions I see in the logs:
```
INFO:vivisect:Function Analysis Exception for function 0x412783, module: vivisect.analysis.generic.nore
INFO:vivisect:Exception Traceback: Traceback (most recent call last):
  File "/path/vivisect/__init__.py", line 781, in analyzeFunction
    fmod.analyzeFunction(self, fva)
  File "/path/vivisect/analysis/generic/noret.py", line 31, in analyzeFunction
    g = v_t_graph.buildFunctionGraph(vw, fva)
  File "/path/vivisect/tools/graphutil.py", line 470, in buildFunctionGraph
    raise Exception("buildFunctionGraph: Attempt to get location at 0x%x" % nextva)
Exception: buildFunctionGraph: Attempt to get location at 0x4127c0
```

### Steps to reproduce

```
> python 3
>>> # Enable logging
>>> import logging
>>> logging.basicConfig(level=logging.INFO)
>>>
>>> import vivisect
>>> file = '9ff8e68343cc29c1036650fc153e69f7.exe_' # https://github.com/fireeye/capa-testfiles/blob/master/9ff8e68343cc29c1036650fc153e69f7.exe_
>>> vw = vivisect.VivWorkspace()
>>> vw.loadFromFile(file)
>>> vw.analyze() # Exception! 
```

**Comment by rakuy0** [comment]
So could you point out what features are missing or different? Maybe not all of them, but if you could point out a couple, that would go a long way in helping us track down what changed. 

But that exception message may (or may not, it depends) be a red herring in that it does happen from time to time in terms when trying to determine if a function is no-return. It is still something we need to fix, but I'd hesitate to say that that message is the definitive cause of the missing features (because that message has appeared in both py2 and py3).

**Comment by Ana06** [comment]
Similarly as with https://github.com/vivisect/vivisect/issues/358, this is also fixed in current master + Py3 Cleanup #353 + Python3 fixes #357.

> but I'd hesitate to say that that message is the definitive cause of the missing features (because that message has appeared in both py2 and py3).

You are right that this message appeared in py2:
```
Function Analysis Exception for 0x4096f8 vivisect.analysis.generic.noret: buildFunctionGraph: Attempt to get location at 0x409791
Function Analysis Exception for 0x42b550 vivisect.analysis.generic.noret: buildFunctionGraph: Attempt to get location at 0x42b5ce
Function Analysis Exception for 0x43d5d0 vivisect.analysis.generic.noret: buildFunctionGraph: Attempt to get location at 0x43d5f3
Function Analysis Exception for 0x409788 vivisect.analysis.generic.noret: buildFunctionGraph: Attempt to get location at 0x409791
Function Analysis Exception for 0x42bb28 vivisect.analysis.generic.noret: buildFunctionGraph: Attempt to get location at 0x42bb2e
Function Analysis Exception for 0x401408 vivisect.analysis.generic.noret: buildFunctionGraph: Attempt to get location at 0x4014c5
Function Analysis Exception for 0x447154 vivisect.analysis.generic.noret: buildFunctionGraph: Attempt to get location at 0x4471e9
```

but in py3 it appears one more time:
`Exception: buildFunctionGraph: Attempt to get location at 0x4127c0`
